components/script/dom/abstractworkerglobalscope.rs

```diff
@@ -68,7 +68,7 @@ impl ScriptChan for WorkerThreadWorkerChan {
 
 impl ScriptPort for Receiver<DedicatedWorkerScriptMsg> {
     fn recv(&self) -> Result<CommonScriptMsg, ()> {
-        let common_msg = match self.recv() {
+        let common_msg = match self.recv().ok() {
```

Comment:
Nit: Remove this `.ok()` and change below:

* `Some` to `Ok`
* `None` to `Err(_)`